tests/pytest.ini

```diff
@@ -0,0 +1,35 @@
+[pytest]
+addopts =
+    -v
+    -s
+    --ff
+    --nf
+    --cov
+    --cov-config=pytest.ini
+    --cov-report term-missing
+python_files =
+    test_*.py
+; -v        run verbose
+; -s        disable all capturing of stdout/stderr```
+; --ff      failed first
+; --nf      new first
+; --cov     show coverage
+; --cov-config=pytest.ini       set coverage config to this file
```

Comment:
This can be placed under the existing ﻿pyproject.toml with the key `[tool.pytest.ini_options]`

Reply:
And coverage settings can go under

`[tool.coverage.report]`

`[tool.coverage.run]`

Reply:
Both pytest and coverage will look for pyproject.toml automatically, no additional config is needed (except for making sure it can be found within the test docker container)